app/Service/StudentDetailsService.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Exceptions\StudentDetailsNotFoundException;
+use App\Models\Resume;
+
+
+class StudentDetailsService
+{
+    public function execute($student)
+    {
+        return $this->getStudentDetailsById($student);
+    }
+    
+
+    public function getStudentDetailsById($student){
+        $studentDetails = Resume::where('student_id', $student)->get();
```

Comment:
You are returning all the data, it is not the structure to be returned. 
Perhaps a misunderstanding :wink:

Reply:
Fixed